devbox.json

```diff
@@ -1,7 +1,7 @@
 {
   "packages": [
-    "golangci-lint@1.52.2",
-    "go@latest"
+    "go@latest",
+    "runx:golangci/golangci-lint@latest"
```

Comment:
Would be preferable to include a source reference. In the future, runx may support github, gitlab, sourcehut, custom domain etc.

```
runx: github.com/golangci/golangci-lint@latest
```